src/Symfony/Component/Notifier/Bridge/FakeChat/Tests/TestLogger.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\FakeChat\Tests;
+
+use Psr\Log\AbstractLogger;
+
+class TestLogger extends AbstractLogger
```

Comment:
Final please

Reply:
ok lets do this

> Please extend UnsupportedSchemeExceptionTest, thanks

I do not understand this comment @OskarStark :s

Reply:
Sorry, please open this file and add a test case for your transport, thanks

Reply:
Does it makes sense to you?